src/components/tabs/tabs-hooks.tsx

```diff
@@ -16,14 +16,15 @@ export const useBuildTabs = (
     let activeIsDisabled: boolean = false
 
     Children.forEach(children, (tab, index) => {
-      if (!tab) return
-      const { props } = tab
```

Comment:
I dont see any difference with the changes you made. Line 19 that you removed returns fast for falsy tabs and do not push children. Can you elaborate more about this change?